plugins/Diagnostics/Commands/Run.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Diagnostics\Commands;
+
+use Piwik\Container\StaticContainer;
+use Piwik\Plugin\ConsoleCommand;
+use Piwik\Plugins\Diagnostics\Diagnostic\DiagnosticResult;
+use Piwik\Plugins\Diagnostics\Diagnostic\DiagnosticResultItem;
+use Piwik\Plugins\Diagnostics\DiagnosticService;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Run the diagnostics.
+ */
+class Run extends ConsoleCommand
+{
+    /**
+     * @var DiagnosticService
+     */
+    private $diagnosticService;
+
+    public function __construct()
+    {
+        // Replace this with dependency injection once available
+        $this->diagnosticService = StaticContainer::get('Piwik\Plugins\Diagnostics\DiagnosticService');
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setName('diagnostics:run')
+            ->setDescription('Run the Piwik diagnostics')
```

Comment:
I think this description could be a bit more descriptive, maybe w/ a quick description of what a Piwik 'diagnostic' is. Or maybe 'Run the Piwik diagnostic checks' w/ a quick description of what they can be used for? If you think it's too much text, maybe it could go in as a help string?